lxd/containers.go

```diff
@@ -757,27 +757,20 @@ func (d *lxdContainer) applyConfig(config map[string]string) error {
 	}
 
 	if lxcConfig, ok := config["raw.lxc"]; ok {
-		for _, line := range strings.Split(lxcConfig, "\n") {
-			contents := strings.SplitN(line, "=", 2)
-			key := contents[0]
-			value := ""
-			if len(contents) > 1 {
-				value = contents[1]
-			}
-
-			key = strings.Trim(key, " ")
-			value = strings.Trim(value, " ")
-
-			// empty lines
-			if key == "" {
-				continue
-			}
+		f, err := ioutil.TempFile("", "tmpConfig")
```

Comment:
Sorry for being picky on that, but my /tmp is usually a mess after a while so I like things prefixed with something I can understand :)

Can you change this to lxd_config or something like that?

Bonus point if you can change the one in images.go too (Serge already took care of those in the client code).